src/include/daos_srv/daos_server.h

```diff
@@ -107,23 +107,18 @@ dss_tls_get()
 		pthread_getspecific(dss_tls_key);
 }
 
-#define D_TIME_START(sp, op)			\
+#define D_TIME_START(sp, start)			\
```

Comment:
even the operation code is not used, I strongly suggest to keep it, because we at least know which operation is profiling, giving you could have multiple operations are profiled in the same function, we don't have to distinguish them by variable names.

Reply:
ok